### ISSUE TITLE
rmdir succeeds without action when a non-directory file is passed

### ISSUE THREAD

**Issue by jstarks** [malxau/yori#93]
Yori 1.40:

```
C:\Users\jostarks\tmp>touch foo
C:\Users\jostarks\tmp>rmdir foo || echo huh?
C:\Users\jostarks\tmp>dir foo

 Directory of C:\Users\jostarks\tmp

03/14/2021  03:26 PM                  0 foo
           1 File(s)                  0 bytes
           0 Dir(s)      22,596,653,056 bytes free
```

**Comment by malxau** [comment]
This was fixed after 1.40 in commit 5de18adb07c5c3c1525f22568c3fbbdd7a28c0f5 .

**Comment by malxau** [comment]
This should be fixed in 1.50, released today.  Please let me know if there are further related issues.
